integration/entity_test.go

```diff
@@ -217,26 +217,35 @@ func Test_Entity_EntitiesWithSameMetadataAreSameAs(t *testing.T) {
 
 func TestEntity_AddCommonDimension(t *testing.T) {
 	tests := []struct {
-		name      string
-		commons   metric.Dimensions
-		expeceted *Entity
+		name     string
+		commons  metric.Dimensions
+		expected *Entity
```

Comment:
what's wrong with `expeceted`??? :trollface:

Reply:
It is not familiar to me, YET 😂